config/channels/in-memory-channel/README.md

```diff
@@ -51,3 +51,20 @@ single Dispatcher for all in-memory Channels.
 ```shell
 kubectl get deployment -n knative-eventing imc-dispatcher
 ```
+
+### Namespace Dispatchers
+
+By default events are received and dispatched by a single cluster-scoped dispatcher components. You can also specify whether events should be received and dispatched by the dispatcher in the same namespace as the channel definition by adding the
+`eventing.knative.dev/scope: namespace` annotation. For instance:
+
+```shell
+kubectl apply --filename - << END
+apiVersion: messaging.knative.dev/v1alpha1
+kind: InMemoryChannel
+metadata:
+  name: foo
```

Comment:
Change name to foo-ns and prevent user to dynamically switch existing channel scope?